EIPS/eip-2159.md

```diff
@@ -0,0 +1,56 @@
+---
+eip: 2159
+title: Common Prometheus Metrics Names for Clients
+author: Adrian Sutton (@ajsutton)
+discussions-to: https://ethereum-magicians.org/t/common-chain-metrics/3415/2
```

Comment:
I suggest to remove the `/2` postfix so that it links to the entire thread.

Reply:
Done.